packages/account-tree-controller/src/index.ts

```diff
@@ -0,0 +1,15 @@
+export type {
+  AccountGroupRoot as AccountTree,
```

Comment:
Why are we renaming these here? Why not name them `AccountTree`, `AccountTreeId`, and `AccountTreeMetadata` in `AccountTreeController.ts`? Or why not stick with these names?

Reply:
I cleaned up all exports, thanks!